hiredis.c

```diff
@@ -122,10 +122,10 @@ static void *createStringObject(const redisReadTask *task, char *str, size_t len
         return NULL;
 
     buf = malloc(len+1);
-    if (buf == NULL) {
+    if (buf == NULL) {          // LCOV_EXCL_START 
```

Comment:
@ashtul Did you consider the tradeoff between more accurate coverage and code clutter ?
Not sure it's worth having LCOV-specific tags all over the place.

Reply:
I see your point
This is really your call
I will remove